release-notes/6.0/supported-os.md

```diff
@@ -43,54 +43,60 @@ Notes:
 OS                              | Version                      | Architectures      | Lifecycle          |
 --------------------------------|------------------------------|--------------------|--------------------|
 [Alpine][6]                     | 3.20, 3.19, 3.18, 3.17       | Arm32, Arm64, x64  | [Lifecycle][7]     |
-[Debian][8]                     | 12, 11                       | Arm32, Arm64, x64  | [Lifecycle][9]     |
-[Fedora][10]                    | 40, 39                       | Arm32, Arm64, x64  | [Lifecycle][11]    |
-[openSUSE Leap][12]             | 15.6, 15.5                   | Arm64, x64         | [Lifecycle][13]    |
-[Red Hat Enterprise Linux][14]  | 9, 8, 7                      | Arm64, x64         | [Lifecycle][15]    |
-[SUSE Enterprise Linux][16]     | 15.5, 12.5                   | Arm64, x64         | [Lifecycle][17]    |
-[Ubuntu][18]                    | 24.04, 23.10, 22.04, 20.04   | Arm32, Arm64, x64  | [Lifecycle][19]    |
+[CentOS][8]                     |                              | x64                | [Lifecycle][9]     |
```

Comment:
@richlander That's more a topic for `distroessed`, but I think it's still good to start the discussion here as that's a nice example.

Should we add a text in the version column if there are no supported versions left? It might confuse people. We could add `[None](#out-of-support-os-versions)` to make it more obvious that the information on why there's nothing can be found in the Out of Support section.

Reply:
That problem occurred to me and your solution is better than what I had considered. Happy to do that.

I'll update .NET 7 as well so that it matches what I did with .NET 6. Naturally, I don't want to keep updating the .NET 7 EOL information, but this change seems warranted to make it correct.